src/cls/rgw/cls_rgw_client.cc

```diff
@@ -624,18 +624,32 @@ int cls_rgw_usage_log_read(IoCtx& io_ctx, string& oid, string& user,
   return 0;
 }
 
-void cls_rgw_usage_log_trim(ObjectWriteOperation& op, string& user,
-                           uint64_t start_epoch, uint64_t end_epoch)
+int cls_rgw_usage_log_trim(IoCtx& io_ctx, const string& oid, string& user,
+			   uint64_t start_epoch, uint64_t end_epoch)
 {
-  bufferlist in;
+  bufferlist in, out;
```

Comment:
`out` looks unused here

Reply:
oops this was used in my first iteration, forgot to remove, dropping